### DISCUSSION TITLE
can i use github wiki - and enable all the visitors to get writing capabilities on the wiki!?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#53375] [New to GitHub]
can i use github wiki - and enable all the visitors to get writing capabilities on the wiki!?

guess that this is possible

look forward to hear form you

**Comment by NJacksonDev** [comment]
You can grant anyone with a GitHub account write access, though by default, GitHub wikis are only editable by users who have write access to the repository that the wiki is associated with. 

However, it is possible to allow anyone with a GitHub account to contribute to your wiki by enabling the "Anyone with write access" option in the repository's settings (see below how to do this).

- Navigate to the repository that your wiki is associated with.

- Click on the "Settings" tab in the navigation bar.

- Scroll down to the "Danger Zone" section and click on the "Collaborators & teams" link.

- In the "Collaborators & teams" section, you'll see a "Repository access" option. Click on the "Add" button next to it.

- In the "Invite collaborators" dialog box, type in the name of the GitHub user you want to invite to your repository.

- Click on the "Write" access dropdown menu and select "Anyone with write access".

- Click on the "Add" button to send the invite.
